rpi-eeprom-update

```diff
@@ -224,16 +224,27 @@ getBootloaderUpdateVersion() {
 }
 
 checkDependencies() {
+   rev="$(sed -n '/^Revision/s/^.*: \(.*\)/\1/p' < /proc/cpuinfo)"
+   if [ $(((0x$rev >> 23) & 1)) -ne 0 ] && [ $(((0x$rev >> 12) & 15)) -eq 3 ]; then
+      echo "BCM2711 detected"
+   else
+      # Not a BCM2711, no EEPROMs to update.
+      exit ${EXIT_SUCCESS}
+   fi
+
    if ! command -v vcgencmd  > /dev/null; then
       die "vcgencmd not found. On Debian, try installing the libraspberrypi-bin package."
    fi
 
-   CPU_VER="$(vcgencmd otp_dump | grep 30: | cut -c8)"
-
-   if [ "${CPU_VER}" != "3" ]; then
-      # Not a BCM2711, no EEPROMs to update.
-      exit ${EXIT_SUCCESS}
-   fi
+   # Exit if VCHI is stuck to avoid blocking apt updates
+   nohup vcgencmd version > /dev/null 2>&1 &
```

Comment:
In the failing case, vcgencmd would return with 'VCHI initialization failed'. I'm not sure that this test does anything.

Reply:
ok, I'll split these changes in-case the otp_dump change is sufficient